projects/storefrontlib/shared/components/card/card.component.html

```diff
@@ -131,46 +131,84 @@
     </div>
     <!-- Actions -->
     <div *ngIf="content.actions && !editMode" class="cx-card-actions">
-      <div *ngFor="let action of content.actions">
+      <div *ngFor="let action of content.actions" class="cx-card-action-item">
         <div>
```

Comment:
Do you think we can get rid of this extra \<div\>, not sure if it serves a purpose? Let's leave it if it's extra work ;)